resolvers.py

```diff
@@ -0,0 +1,1979 @@
+            nested_data.append(self.process_account_data(
+                account_name, account_data))
+
+        return nested_data
+    
+class WBSTrackerAccountsResolver:
+
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        qs = models.WbsTrackingFact.objects.all().values('account_name',
+                                                         'billability').distinct('account_name').filter(~Q(billability='Total'))
+        return list(qs)
+
+
+class BillingWBSCodePopUpResolver:
+
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        try:
+            pod_id = kwargs.get("pod_id")
+            qs = models.WbsTrackingFact.objects.filter(
+                pod_id=pod_id).values_list('pod_id', 'active_pod_wbs_codes')
+            result = {"pod_id": pod_id}
+            children = []
+            for item in list(qs)[0][1]:
+                value = {}
+                value["wbs_code"] = item["wbs_code"]
+                value["percentage"] = item["allocation_amount_percentage"]
+                children.append(value)
+            result["active_pod_details"] = children
+
+            return result
+
+        except Exception:
+            raise PodIdNotFound("POD-ID does not exist")
+
+
+class DistinctFieldResolver:
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        try:
+            model_name = kwargs.get("model_name")
+            columns_list = kwargs.get("column_list")
+            final_result = {}
+            model_instance = getattr(models, model_name)
+
+            for column in columns_list:
+
+                field_data = model_instance.objects.values_list(
+                    column, flat=True).distinct().order_by(column)
+                final_result[column] = list(field_data)
+
+            data = json.loads(json.dumps(final_result, cls=DjangoJSONEncoder))
+            response_data = {"ok": True, "error": "", "data": data}
+
+            return response_data
+
+        except (AttributeError, FieldError) as e:
+            return Exception(e)
+
+
+class GetAllFieldsOfModelResolver:
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        try:
+            model_name = kwargs.get("model_name")
+
+            model_instance = getattr(models, model_name)
+
+            list_of_columns = model_instance._meta.get_fields()
+
+            response = [{"field_name": []}]
+
+            for values in list_of_columns:
+                response[0]['field_name'].append(values.name)
+
+            return response
+        except (AttributeError):
+            return Exception("Model does not exists")
+
+class ReportTemplateHelper:
+    def apply_selected_column(self,column_list):
+
+        qs = models.MasterBaseFact.objects.values(*column_list)
+
+        return qs
+
+    def apply_filter_on_column(self, qs, filter_list):
+
+        if filter_list == None:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)